client/src/locales/en-US.json

```
@@ -1567,18 +1567,19 @@
             "admin": "Administrators | Administrator | Administrators",
             "standard": "Members | Member | Members",
             "outsider": "Externals | External | Externals",
-            "consumptionDetail": "Consumption detail",
-            "nonPaying": "Non paying",
-            "paying": "charged",
-            "free": "free",
-            "ofWhich": "of which",
-            "data": "data",
-            "metadata": "metadata",
+            "total" : "Total",
+            "detail": "Detail",
+            "consumption": "Consumption",
```

It might depend on the exact context, but this is more likely referring to "Usage".
If that's the case, I suggest changing the key too. 
```suggestion
            "usage": "Usage",
```